pom.xml

```diff
@@ -100,9 +102,12 @@
 		<codahale.metrics.version>3.0.0</codahale.metrics.version>
 		<avro.version>1.7.6</avro.version>
 		<jets3t.version>0.7.1</jets3t.version>
+                <commons.math3.version>3.2</commons.math3.version>
+                <commons.httpclient.version>4.3.6</commons.httpclient.version>
+                <io.netty.version>4.0.17.Final</io.netty.version>
```

Comment:
@xhumanoid 
Thanks for reporting problem. Zeppelin enables spark-1.2.0+ support by setting maven profile 'spark-1.2'. Profile 'spark-1.2' overrides io.netty.version to '4.0.23.Final' (line 1223). So, i think it's okay with 'spark-1.2' profile.